app/components/search/SearchMap.jsx

```diff
@@ -84,17 +84,20 @@ const SearchMap = ({
     return null;
   }
 
-  const markers = hits.map((hit, index) => (
-    <CustomMarker
-      hit={hit}
-      page={page}
-      hitsPerPage={hitsPerPage}
-      lat={hit._geoloc ? hit._geoloc.lat : 0}
-      lng={hit._geoloc ? hit._geoloc.lng : 0}
-      key={hit.objectID}
-      index={index}
-    />
-  ));
+  const markers = hits.map((hit, index) => {
+    const { addresses = [] } = hit;
+    return addresses.length ? addresses.map(address => (
```

Comment:
This `address.length` check is unnecessary. `Array.prototype.map()` will already return an empty array for an empty array input.